src/main/java/frc/robot/shuffleboard/ShuffleboardWidgets.java

```diff
@@ -19,42 +19,63 @@
 
   ShuffleboardTab tab;
 
-  /*private NetworkTableEntry P_testing;
-  private NetworkTableEntry I_testing;
-  private NetworkTableEntry D_testing;
-  double P_Value = 0.0;
-  double I_Value = 0.0;
-  double D_Value = 0.0;*/
+  /*
```

Comment:
Code Hygiene: Delete these commented line.

Reply:
It could be of use when we do PID testing thus i kept it